client.go

```diff
@@ -2119,3 +2166,11 @@ func (fs *FileStat) IsDir() bool {
 func (fs *FileStat) Sys() interface{} {
 	return fs
 }
+
+type FileSecurityInfo struct {
+	Owner *Sid
```

Comment:
Sid is internal type, we should avoid exposing it. maybe string representation is enough for SIDs